### PR TITLE
Cap maximum connection retries

### PR DESCRIPTION
**Pull request by Prinsn**
Sets maximum retries, currently to 10, and then sets connection status to failed

TODO: 

Currently websocket keeps receiving error messages, despite the close command
Should likely change from "Failed" to something else at some point

### CODE REVIEW

File: .gitignore
Comment (AutomaticFrenzy) [inline .gitignore:1]: can shorten it to just one line with /external/SDL2_mixer/external/mpg123-1.25.6/ports/MSVC++/2008clr/examples/ to cover everything inside

File: external/APCpp/Archipelago.cpp [hunk 1]
```diff
@@ -23,11 +23,14 @@
 constexpr int AP_OFFLINE_SLOT = 1404;
 #define AP_OFFLINE_NAME "You"
 
+const int MAX_RETRIES = 1;
```
Comment (AutomaticFrenzy) [inline external/APCpp/Archipelago.cpp]: lets default to 3 incase of an initial network hiccup

**Review by AutomaticFrenzy — COMMENTED** [review]
we should probably include being able to interrupt the connection attempt with escape or something, but that doesn't have to be in this pr

**Comment by Prinsn** [comment]
I intended 10, 1 was for quick testing because it takes longer and longer
reach retry

6 should be minimum if 3 because it has a fail over after 3 to retry
without ssh

**Comment by AutomaticFrenzy** [comment]
threw a commit on, looks good to me

**Pull request merged**